datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -143,6 +150,17 @@ export default function UserInfoSideBar({ sideBarData, refetch }: Props) {
                         </Space>
                     </SocialDetails>
                     <Divider className="divider-aboutSection" />
+                    { countryCode && (
```

Comment:
Try to avoid using && inside of rendering code. This results in a "0" being printed in some false cases.

Instead fo:

countryCode ?

.... : null